email_template_qweb/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "QWeb for email templates",
+    "version": "9.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Marketing",
+    "summary": "Use the QWeb templating mechanism for emails",
+    "depends": [
+        'mail',
+    ],
+    "demo": [
+        "demo/ir_ui_view.xml",
+        "demo/mail_template.xml",
+    ],
+    "data": [
+        "views/mail_template.xml",
+    ],
+    "installable": True,
+    "application": False,
```

Comment:
Honestly I just don't know the default values, so I define them. Are we good to just leave these out?